Doc/c-api/gcsupport.rst

```diff
@@ -60,6 +60,21 @@ Constructors for container types must conform to two rules:
    followed by the :c:member:`~PyTypeObject.tp_traverse` handler become valid, usually near the
    end of the constructor.
 
+
+.. c:function:: int PyObject_IS_GC(PyObject *obj)
```

Comment:
I'm not fully satisfied by the documentation. When I read it, the intent of the function is not obvious :-(

I propose:

```
Test if an object has the :c:type:`PyGC_Head` header
(if it can be tracked by the garbage collector).
```

Reply:
I discussed the documentation of this function with @pablogsal. He dislikes the documentation that I proposed: it exposes too many implementation details. He prefers a way simpler documentation. So I propose to replace the whole documentation with just:

> Returns non-zero if the object implements the garbage collector protocol, otherwise returns 0.
> 
> The object cannot be tracked by the garbage collector if this function returns 0.

Reply:
Thanks, I learned much presentation skill in here ;)